data/sql/updates/pending_db_world/rev_1688557801143854400.sql

```diff
@@ -0,0 +1,6 @@
+-- Vile Like Fire! (13071)
+-- fix Njorndar Proto-Drake vehicle unable to fly
+INSERT INTO `acore_world`.`creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES (30564, 6, 57403, 12340);
+
+-- fix Njorndar Proto-Drake display status on the ground
+UPDATE `acore_world`.`creature_template_movement` SET `Flight`=2 WHERE  `CreatureId`=30272;
```

Comment:
Please, insert DELETE before INSERT following the [AC sql standard](https://www.azerothcore.org/wiki/sql-standards#insert-amp-delete). Moreover, you don't need to put the name of the database, because it can change client by client, it is important to put only the table name.
```suggestion
DELETE FROM `creature_template_spell` WHERE `CreatureID` = 30564;
INSERT INTO `creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES (30564, 6, 57403, 12340);

-- fix Njorndar Proto-Drake display status on the ground
UPDATE `creature_template_movement` SET `Flight`=2 WHERE  `CreatureId`=30272;
```

Thank you for your contribution btw 👍 🚀

Reply:
has been updated, thanks for your instruction.